src/atoms/icons/iconWrapper.tsx

```diff
@@ -8,6 +8,7 @@ const IconComponent = Styled.div<IconWrapperProps>`
   width: ${(props) => props.width};
   display: flex;
   align-items: center;
+  justify-content: center;
```

Comment:
@shan8851 is it safe change ?

Reply:
Hey @goshatravin yes, basically since we changed icon component we are using svg with plate and spacing exactly as Michal has it in design, but this means our container is 24 x 24, but the svg inside is sometimes off, so see in image that icon is off centre inside container.

![Screenshot 2022-11-22 at 18 15 30](https://user-images.githubusercontent.com/39712238/203390662-f87ceb13-9fae-416a-bca1-e6f3d4607c49.png)

So this just puts it in centre

![Screenshot 2022-11-22 at 18 16 00](https://user-images.githubusercontent.com/39712238/203390740-3cc4aff0-d9c4-4838-8c69-6c4830ecf054.png)